firmware/gameport-adapter/gameport-adapter.ino

```diff
@@ -18,6 +18,7 @@
 #include "HidJoystick.h"
 
 #include "CHFlightstickPro.h"
+#include "CH-f16-combat-stick.h"
```

Comment:
Would be nice, if you'd follow the other naming scheme. The file should be named just as the class inside `CHF16CombatStick.h`